pkg/pb/plan/plan.pb.go

```diff
@@ -866,7 +866,6 @@ func (m *Type) GetTable() string {
 type Const struct {
 	Isnull bool `protobuf:"varint,1,opt,name=isnull,proto3" json:"isnull,omitempty"`
 	// Types that are valid to be assigned to Value:
-	//
```

Comment:
Please check whether your protobuf tools are at the newest version

Reply:
when i use "make pb" in MO, 
i will get the same xx.pb.go  (compare the origin pb.go file).
i think my version is fine.